src/main/java/at/hannibal2/skyhanni/features/garden/FarmingFortuneDisplay.kt

```diff
@@ -53,7 +54,7 @@ object FarmingFortuneDisplay {
     )
     private val tooltipFortunePattern by patternGroup.pattern(
         "tooltip",
```

Comment:
```suggestion
        "tooltip.new",
```

so that people on old versions do not get errors with the new pattern that was added before